src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -76,14 +76,21 @@ public void DisabledProtocols_SetSslProtocols_ThrowsException(SslProtocols disab
         }
 
         [ConditionalTheory(nameof(BackendSupportsSslConfiguration))]
-        [InlineData(SslProtocols.Tls)]
-        [InlineData(SslProtocols.Tls11)]
-        [InlineData(SslProtocols.Tls12)]
-        public async Task GetAsync_AllowedSSLVersion_Succeeds(SslProtocols acceptedProtocol)
+        [InlineData(SslProtocols.Tls, false)]
+        [InlineData(SslProtocols.Tls, true)]
+        [InlineData(SslProtocols.Tls11, true)]
+        [InlineData(SslProtocols.Tls11, false)]
```

Comment:
Nit: Why is the order `true` then `false` for `Tls11`, but `false` then `true` for `Tls` above and `Tls12` below? I found it harder to read this way vs. all following a `false` then `true` order.